### PR TITLE
Added MacOS build details to build in a ext4 image 

### PR DESCRIPTION
**Pull request by psychowood**
Addresses #7

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,17 +50,46 @@ After the build, the images will be inside `~/mybuild/openwrt/bin/target/`.
...
+Create a sparse disk image (to limit space usage):
+```zsh
+mkdir ~/Documents/openwrt
+cd ~/Documents/openwrt
+hdiutil create -size 64g -fs "Case-sensitive HFS+" -type SPARSEBUNDLE -volname openwrt-dev-env openwrt-dev-env.dmg 
+hdiutil attach openwrt-dev-env.dmg
 ```
 
 Then run:
+```zsh
```
Comment (mwarning) [inline README.md:62]: zsh? :-P

Comment (psychowood) [reply README.md:62]: It's the default shell in MacOS :P

Comment (mwarning) [reply README.md:62]: Why do you use `shell` in the other lines then?

Comment (psychowood) [reply README.md:62]: Well the other lines are either for the Linux part or run inside the debian container. Excess of zeal,  I suppose :)

Comment (mwarning) [reply README.md:62]: ok, got it. :-)

**Pull request merged**